libs/logger/inc/public/tfc/logger.hpp

```diff
@@ -1,9 +1,15 @@
 #pragma once
 
+// Standard
+#include <memory>
```

Comment:
memory is included below.
you can do `git checkout main libs/logger/inc/public/tfc/logger.hpp` in git root to revert changes to this file